exporter/exporter.go

```diff
@@ -0,0 +1,94 @@
+package exporter
+
+import (
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/vitwit/avail-monitor/config"
+	"github.com/vitwit/avail-monitor/monitor"
+)
+
+type availCollector struct {
+	config           *config.Config
+	nodeVersion      *prometheus.Desc
+	chainName        *prometheus.Desc
+	councilMember    *prometheus.Desc
+	electedMember    *prometheus.Desc
+	currentValidator *prometheus.Desc
+}
+
+func NewAvailCollector(cfg *config.Config) *availCollector {
+	return &availCollector{
+		config: cfg,
+		nodeVersion: prometheus.NewDesc(
+			"avail_monitor_chain_node_version",
+			"Node Version Information",
+			[]string{"version"}, nil),
+
+		chainName: prometheus.NewDesc(
+			"avail_monitor_chain_name",
+			"Name of the chain",
+			[]string{"chain"}, nil),
+
+		councilMember: prometheus.NewDesc(
+			"avail_monitor_council_members",
+			"council members of the network",
+			[]string{"value"}, nil),
+
+		electedMember: prometheus.NewDesc(
+			"avail_monitor_elections_member",
+			"elected members of the network",
+			[]string{"value"}, nil),
+
+		currentValidator: prometheus.NewDesc(
```

Comment:
Instead of exporting the pubkeys of the validators let's just stick with the number of validators metrics mentioned in the doc